### ISSUE TITLE
[BUG] Android: Unable to run the project in debug mode

### ISSUE THREAD

**Issue by dobromirdobrev** [rokwire/illinois-app#3957]
**Describe the bug**
We are not able to run the UIUC (Illinois app) project for Android in debug mode. This happens after the latest merge and more specifically the problem comes from the flutter plugin `flutter_pdfview` https://pub.dev/packages/flutter_pdfview . 
There is no such problem if we exclude the plugin from the build.

**To Reproduce**
1. Try to run the project in debug mode using Android studio or
2. Use the command in the terminal:
```
flutter build apk --debug --no-tree-shake-icons -t lib/mainDev.dart --flavor IllinoisDev
```

**Expected behavior**
Run the project in debug mode without problems

**Additional context**
Stacktrace:
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:mergeIllinoisDevDebugNativeLibs'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.MergeNativeLibsTask$MergeNativeLibsTaskWorkAction
   > 2 files found with path 'lib/arm64-v8a/libc++_shared.so' from inputs:
      - /.gradle/caches/transforms-3/b089a7e6b427a2938547a133c309c9b6/transformed/jetified-arm64_v8a_debug-1.0.0-74d16627b940bb15e50891f82cad6c3e3465bd6d/arm64-v8a/libc++_shared.so
      - /.gradle/caches/transforms-3/c08aef396bb99c9bf0d90161072dbcc7/transformed/jetified-pdfium-android-1.9.2/jni/arm64-v8a/libc++_shared.so
     If you are using jniLibs and CMake IMPORTED targets, see
     https://developer.android.com/r/tools/jniLibs-vs-imported-targets

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 19s
Running Gradle task 'assembleIllinoisDevDebug'...                  19.8s
Gradle task assembleIllinoisDevDebug failed with exit code 1
```

**Comment by shurwit** [comment]
Linking related issue on library: https://github.com/endigo/flutter_pdfview/issues/264

**Comment by roberlander2** [comment]
> Linking related issue on library: [endigo/flutter_pdfview#264](https://github.com/endigo/flutter_pdfview/issues/264)

According to the last comment in this issue, the problem should be resolved in Flutter 3.16.2.

**Comment by dobromirdobrev** [comment]
I'm closing this issue as the workaround seem to work. Once we update the flutter to latest version, we should verify if the build works without it. Thanks
